uint16 value
---
int32 status
string message
